### DISCUSSION TITLE
[5.0] Feature request: Enable PostDate and ExpiryDate for Entry Types

### DISCUSSION THREAD

**Discussion by Megafry** [craftcms/cms#14358] [Ideas]
## Problem Statement
Entry types can be enabled and disabled, but there is no option to set PostDate or ExpiryDate.
This is only possible when you create a section from an entry type.
In the edit view, the fields are not visible, this is reserved for sections only.

| Sections    | Entry types |
| -------- | ------- |
| ![image](https://github.com/craftcms/cms/assets/10438940/75f61c1c-d042-4d43-91e9-9d0076d8b4e5)   | ![image](https://github.com/craftcms/cms/assets/10438940/26790d76-6652-4537-ac97-220b1ef7e70f)   |

By manually setting an old date to an entry type via the database table entries, it disappears from the frontend output.
Therefore the logic is apparently already implemented.
Also, the status indicator works. (see picture)
![image](https://github.com/craftcms/cms/assets/10438940/c5bfd43b-906c-4ed6-87e4-e405365be2d9)

## Proposed Solution
Make the field also visible for entry Types to give users control over the publication and expiration schedule.

it would be a plus if the start and end date could be added directly in the dropdown menu of matrix field.
ex:
![image](https://github.com/craftcms/cms/assets/10438940/1beaa071-9abd-40da-8994-25dcda40d6fc)

## Benefit
Users can schedule entries to be published at specific times or hide them after a certain period, improving content management flexibility.

**Comment by wmdhosting** [comment]
that exist for years..10+ as much i know..

**Reply by thupsi** [reply]
Maybe the OP means entries inside matrix fields?

**Reply by Megafry** [reply]
My bad, I was referring to entry types.
Under the hood, all entries (section and entry types) are stored in the `entries` table.
I just updated my description accordingly.

Yes, therefore I thought of matrix fields.

**Comment by alexanderloewe** [comment]
+1
